SECURITY_RELEASE_PROCESS.md

```diff
@@ -291,14 +296,33 @@ We accept.
 
 We are definitely willing to help!
 
-> 8. Have someone already on the list vouch for the person requesting membership
-     on behalf of your distribution.
+> 8. Be able to perform a security release of your product within a two week window from candidate fix
+     patch availability.
+
+We affirm we can spin out new security releases within a 2 week window.
+
+> 9. Have someone already on the list vouch for the person requesting membership
+>    on behalf of your distribution.
 
 CrashOverride will vouch for the "Seven" distribution joining the distribution list.
 
-> 9. Nominate an e-mail alias or list for your organization to receive updates. This should not be
-     an individual user address, but instead a list that can be maintained by your organization as
-     individuals come and go. A good example is envoy-security@seven.com, a bad example is
-     acidburn@seven.com. You must accept the invite sent to this address or you will not receive any
-     e-mail updates.
+10. Nominate an e-mail alias or list for your organization to receive updates. This should not be
```

Comment:
Not sure, but I think this line is meant to start with a `>`, and the indentation for following lines preserved.